superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -993,15 +1004,15 @@ export const fields = {
   },
 
   viewport_latitude: {
-    type: 'IntegerField',
+    type: 'TextField',
```

Comment:
one reason i was using a type called `IntegerField` was so we could do number validation on the field in that component, but we could handle this in a different way in the future.